lib/services/northBound/deviceProvisioningServer.js

```diff
@@ -65,6 +66,15 @@ var async = require('async'),
         explicitAttrs: 'explicitAttrs'
     };
 
+
+if (config.relaxTemplateValidation) {
```

Comment:
Functionality added by this PR (along with a mention to the new env var) should be added to CHANGES_NEXT_RELEASE

Reply:
Fixed cb1401e